### PR TITLE
Testing setup

### PR DESCRIPTION
**Pull request by Fleurtam**
#### Description
Setup Testing environment
* Added rspec
* Added cucumber
#### What I have learned working on this feature: 
That we always need to db:migrate, and bundle exec rails generate rspec:install

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -0,0 +1 @@
+--require spec_helper
```
Comment (holgertidemand) [inline .rspec:1]: Add ``` --format documentation```

**Review by holgertidemand — CHANGES_REQUESTED** [review]

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
```
Comment (jyuw) [inline spec/spec_helper.rb]: You can probably get rid of the comments in here

File: .rspec (hunk 1 above)
Comment (holgertidemand) [reply .rspec:1]: ``` --color ```

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
```
Comment (jyuw) [inline spec/rails_helper.rb]: You could get rid of the comments in this file

File: Gemfile [hunk 4]
```diff
@@ -15,4 +15,15 @@ group :development, :test do
   gem 'factory_bot_rails'
   gem 'pry-byebug'
   gem 'pry-rails'
+  gem 'rspec-rails'
+  gem 'shoulda-matchers'
+  gem 'cucumber-rails', require: false
+  gem 'database_cleaner'
```
Comment (holgertidemand) [inline Gemfile:21]: Add ``` gem 'launchy' ```

Comment (Fleurtam) [reply Gemfile:21]: did that!

**Review by magnus-thor — APPROVED** [review]
overall looks fine, but would like to see some git-pong and more detailed commit messages.

**Pull request merged**